src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php

```diff
@@ -388,11 +388,12 @@ public function optionsJson($uri, array $data = [], array $headers = [])
      * @param  string  $uri
      * @param  array  $data
      * @param  array  $headers
-     * @return \Illuminate\Foundation\Testing\TestResponse
+     * @param  bool  $encryptCookies
+     * @return  \Illuminate\Foundation\Testing\TestResponse
      */
-    public function json($method, $uri, array $data = [], array $headers = [])
+    public function json($method, $uri, array $data = [], array $headers = [], bool $encryptCookies = false)
```

Comment:
I just wouldn't add this `$encryptCookies` boolean here, it feels a bit inconsistent with the other methods and there's the `disableCookieEncryption` method for that already